### ISSUE TITLE
On iPhone when Voiceover is ON swipe focus is not set correctly on chart data points or graphs

### ISSUE THREAD

**Issue by RashmiBal** [highcharts/highcharts#15798]
#### Expected behaviour
Voiceover focus should be correctly set exactly over the images when we navigate on the graphs/data points of chart

#### Actual behaviour
Voiceover focus is incorrect, is not set exactly over the image but displaced, when we navigate on the graphs/data points of chart when we swipe

#### Live demo with steps to reproduce
On iPhone go to Funnel chart - https://www.highcharts.com/demo/funnel
Turn on the Voiceover screenreader
Swipe over charts 
https://media.giphy.com/media/t0kodTEvl8tIBWrxj3/giphy.gif

#### Product version

Latest for this page - https://www.highcharts.com/demo/funnel

#### Affected browser(s)
Safari, iOS, iPhone, Voiceover screenreader

**Comment by KacperMadej** [comment]
Hi @RashmiBal ,

Thank you for reporting the problem.

**Comment by oysteinmoseng** [comment]
Hi @RashmiBal, this is unfortunately a problem with VoiceOver and how it handles positioning of SVG elements. Not much we can do about this, it would have to be taken up with the VoiceOver developers.

I am closing this for now, but feel free to comment and we can reopen if there is anything we should handle on our end.
